settings.py

```diff
@@ -1,2 +1,11 @@
-client_id = 'Your client_id from https://stepic.org/oauth2/applications/'
-client_secret = 'Your client_secret from https://stepic.org/oauth2/applications/'
+"""
+client_id - 'Your client_id from https://stepic.org/oauth2/applications/'
+client_secret - 'Your client_secret from https://stepic.org/oauth2/applications/'
+"""
+
+import os
+
+client_id = os.environ.get('STEPIC_CLIENT_ID')
+client_secret = os.environ.get('STEPIC_CLIENT_SECRET')
```

Comment:
Simply, I don't like API keys in the code. In this project it's safe even in case you upload it.

Ok, I will revert it back.